src/common/runtime/server.ts

```diff
@@ -71,13 +71,13 @@ if (!sys.existsSync('src/common/runtime/cmdline.ts')) {
   console.log('Must be run from repository root');
   usage(1);
 }
+setBaseResourcePath('out-node/resources');
```

Comment:
Can this read the cache from `src` instead of `out-node`?

Reply:
It could, yes, but it seems a lot cleaner to have all the build under one directory for serving.

Reply:
Ah, if the server runtime is dependent on out-node already then SGTM.